src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -68,33 +93,46 @@ export const ChainSwitcher = ({ isMobile }: ChainSwitcherProps) => {
   const selectChain = useSelectChain()
   useSyncChainQuery()
 
-  if (!chainId || !info) {
-    return null
-  }
-
   return (
     <Box position="relative" ref={ref}>
-      <Row as="button" gap="8" className={styles.ChainSwitcher} onClick={toggleOpen}>
-        <img src={info.logoUrl} alt={info.label} className={styles.Image} />
-        <Box as="span" className={subhead} style={{ lineHeight: '20px' }}>
-          {info.label}
-        </Box>
+      <Row
+        as="button"
+        gap="8"
+        className={styles.ChainSwitcher}
+        background={isOpen ? 'accentActiveSoft' : 'none'}
+        onClick={toggleOpen}
+      >
+        {!chainId || !info ? (
+          <>
+            <TokenWarningRedIcon fill={themeVars.colors.darkGray} width={24} height={24} />
+            <Box as="span" className={subhead} style={{ lineHeight: '20px' }}>
+              {info?.label ?? 'Unsupported'}
+            </Box>
+          </>
```

Comment:
If no chain id is provided, that would mean there's no network connection... Is `Unsupported` the right language for that scenario?

Reply:
That would be a question for @infredible . Unsupported is the language used in the Figma

Reply:
it's more than `chainId` being undefined means web3-react is still loading most likely. as long as that case is handled unsupported won't be necessary. rather, you'll need to use the `isChainSupported` helper to check if supported chainId

Reply:
Thanks for the clarification, I believe the helper fn is named `isChainAllowed`